src/applications/caregivers/components/SubmitError/DownloadLink.jsx

```diff
@@ -1,48 +1,55 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
+import * as Sentry from '@sentry/browser';
 import moment from 'moment';
 
 import formConfig from 'applications/caregivers/config/form';
 import environment from 'platform/utilities/environment';
+import recordEvent from 'platform/monitoring/record-event';
 import { submitTransform } from 'applications/caregivers/helpers';
 import { apiRequest } from 'platform/utilities/api';
+import { isValidForm } from 'platform/forms-system/src/js/validation';
+import { createFormPageList } from 'platform/forms-system/src/js/helpers';
 
 const DownLoadLink = ({ form }) => {
   const [PDFLink, setPDFLink] = useState(null);
   const formData = submitTransform(formConfig, form);
   const veteranFullName = form.data.veteranFullName;
+  const pageList = createFormPageList(formConfig);
+  const isFormValid = isValidForm(form, pageList);
 
-  useEffect(
-    () => {
-      apiRequest(
-        `${environment.API_URL}/v0/caregivers_assistance_claims/download_pdf`,
-        {
-          method: 'POST',
-          body: formData,
-          headers: {
-            'Content-Type': 'application/json',
-            'Source-App-Name': 'caregivers-10-10cg-',
-          },
+  const downloadPDF = () => {
+    apiRequest(
+      `${environment.API_URL}/v0/caregivers_assistance_claims/download_pdf`,
+      {
+        method: 'POST',
+        body: formData,
+        headers: {
+          'Content-Type': 'application/json',
+          'Source-App-Name': 'caregivers-10-10cg-',
         },
-      )
-        .then(response => {
-          return response.blob();
-        })
-        .then(blob => {
-          const url = URL.createObjectURL(blob);
-          setPDFLink(url);
-        });
-    },
-    [formData],
-  );
+      },
+    )
+      .then(response => response.blob())
+      .then(blob => {
+        const url = URL.createObjectURL(blob);
+        setPDFLink(url);
+        recordEvent({ event: 'caregivers-10-10cg-pdf-download--success' });
+      })
+      .catch(error => {
+        Sentry.withScope(scope => scope.setExtra('error', error));
```

Comment:
This was intentional since we are catching the error its no longer going to reach sentry so wanted to have GTM and sentry logging for this important error